l10n_co_geo/data/res_country_state.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nice idea, but as far as I can see, ther is no CO.zip (http://download.geonames.org/export/zip/) Or is it inlcuded in the allCountries.zip? In any case, we still would need to include the DIAN code which is not any of ISO or ZIP. Any ideas of how we could integrate this? Including feed information back into geonames.org? Especially, if we could build a base for Colombian ZIP code this would be cool, as it is barely used yet.